### PR TITLE
[Spring] Add examples to the documentation

### PR DESCRIPTION
**Pull request by sebastienrussell**
## Summary

I try for almost 2-3 days to use the spring injection of bean in my cucumber scenarios, but the documentation was leading me in the wrong direction.

## Details

I added examples in the documentation of Spring so that others don't waste their times.

## Motivation and Context

I really thinks that this change can improve readability of your documentation.

## How Has This Been Tested?

I only did a change in documentation, so tests are not needed I think.

## Types of changes

No change other than a documentation with exemples

## Checklist:

- [ ] I've added tests for my code.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: spring/README.md [hunk 1]
```diff
@@ -23,6 +23,24 @@ Add the `cucumber-spring` dependency to your `pom.xml`:
 For your own classes:
 
 * Add a `@Component` annotation to each of the classes cucumber-spring should manage.
+```java
+package cucumber.runtime.java.spring.beans;
```
Comment (mpkorstanje) [inline spring/README.md]: Please use the `com.example.app` package

File: spring/README.md [hunk 2]
```diff
@@ -32,13 +50,34 @@ import org.springframework.context.annotation.ComponentScan;
 import org.springframework.context.annotation.Configuration;
 
 @Configuration
-@ComponentScan("com.example.app")
+@ComponentScan("cucumber.runtime.java.spring.beans")
```
Comment (mpkorstanje) [inline spring/README.md]: This should not be changed. The `com.example.app` should be used for consistency everywhere.

File: spring/README.md [hunk 3]
```diff
@@ -32,13 +50,34 @@ import org.springframework.context.annotation.ComponentScan;
...
 public class Config {
     // the rest of your configuration
 }
 ```
 
 * Add `@DirtiesContext` to your test configuration if each scenario should have a fresh application context.
+```java
+package cucumber.runtime.java.spring.dirtiescontextconfig;
```
Comment (mpkorstanje) [inline spring/README.md]: `com.example.app`

File: spring/README.md [hunk 4]
```diff
@@ -32,13 +50,34 @@ import org.springframework.context.annotation.ComponentScan;
...
+import com.example.app.Config;
+import cucumber.runtime.java.spring.beans.Belly;
+import cucumber.runtime.java.spring.beans.BellyBean;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.ContextConfiguration;
+
+@ContextConfiguration(classes = { Config.class })
+@DirtiesContext
+public class DirtiesContextBellyStepDefs {
```
Comment (mpkorstanje) [inline spring/README.md]: To keep the documentation consistent this whole class should be similar to `SomeServiceSteps` with ` @DirtiesContext` added.

**Comment by aslakhellesoy** [comment]
Hi @sebastienrussell,

Thanks for your making your first contribution to Cucumber, and welcome to the Cucumber committers team! You can now push directly to this repo and all other repos under the cucumber organization! 🍾

In return for this generous offer we hope you will:

* ✅ Continue to use branches and pull requests. When someone on the core team approves a pull request (yours or someone else's), you're welcome to merge it yourself.
* 💚 Commit to setting a good example by following and upholding our [code of conduct](https://cucumber.io/conduct) in your interactions with other collaborators and users.
* 💬 Join the [community Slack channel](http://cucumber.io/support#slack) to meet the rest of the team and make yourself at home.
* ℹ️ Don't feel obliged to help, just do what you can if you have the time and the energy.
* 🙋 Ask if you need anything. We're looking for feedback about how to make the project more welcoming, so please tell us!

On behalf of the Cucumber core team,
Aslak Hellesøy
Creator of Cucumber

**Comment by mpkorstanje** [comment]
@sebastienrussell didn't realize you fixed the comments a while ago.  Cheers!

**Pull request merged**